### PR TITLE
Change slide 20 caption

### PR DESCRIPTION
**Pull request by conradlotz**
This pull request adds a caption to the 20th slide of the slide deck.

👍

### CODE REVIEW

File: _posts/0000-01-21-JLC007.md [hunk 1]
```diff
@@ -6,4 +6,4 @@ title: "Welcome to our slide deck!"
 ![evil-raccooon](https://cloud.githubusercontent.com/assets/16547949/25400793/b1104eee-29c1-11e7-89e2-3c99f9ea1941.jpg)
 {: .center}
 
-Caption here
+Yes, Yes...I got you all where I want you!
```
Comment (githubteacher) [inline _posts/0000-01-21-JLC007.md:9]: @JLC007 how about adding a second caption?

**Review by githubteacher — CHANGES_REQUESTED** [review]

File: _posts/0000-01-21-JLC007.md (hunk 1 above)
Comment (conradlotz) [reply _posts/0000-01-21-JLC007.md:9]: ok! 👍

File: _posts/0000-01-21-JLC007.md [hunk 2]
```diff
@@ -6,4 +6,4 @@ title: "Welcome to our slide deck!"
 ![evil-raccooon](https://cloud.githubusercontent.com/assets/16547949/25400793/b1104eee-29c1-11e7-89e2-3c99f9ea1941.jpg)
 {: .center}
 
-Caption here
```
Comment (zaheer-d) [inline _posts/0000-01-21-JLC007.md:9]: @JLC007 what was wrong with the original text?

**Review by zaheer-d — COMMENTED** [review]
another revew

File: _posts/0000-01-21-JLC007.md (hunk 2 above)
Comment (conradlotz) [reply _posts/0000-01-21-JLC007.md:9]: cool

**Review by zaheer-d — APPROVED** [review]
this is worthy!

**Review by githubteacher — APPROVED** [review]

**Pull request merged**